modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```
@@ -106,6 +110,13 @@ if (${TFM_PARTITION_CRYPTO})
        endif()
    endif()
 
+  target_link_libraries(platform_crypto_keys
+    PUBLIC
+      platform_s
+      # Link with tfm_sprt to get the include path for tfm_sp_log.h
+      tfm_sprt
```

```suggestion
    PUBLIC
    platform_s
    # Link with tfm_sprt to get the include path for tfm_sp_log.h
    tfm_sprt
```

Keep in-fitting with other parts of the file